apps/condo/domains/news/tasks/notifyResidentsAboutNewsItem.spec.js

```diff
@@ -24,6 +25,8 @@ const { createTestProperty } = require('@condo/domains/property/utils/testSchema
 const { createTestResident } = require('@condo/domains/resident/utils/testSchema')
 const { makeClientWithResidentUser } = require('@condo/domains/user/utils/testSchema')
 
+const TITLE_MAX_LEN = 50
+const BODY_MAX_LEN = 150
```

Comment:
I think it is not worth duplicating these constants, it is better to import them